test/e2e/package.json

```diff
@@ -20,6 +20,7 @@
 		"clean:output": "rm -rf ./allure-results && rm -rf ./results",
 		"build": "yarn workspace @automattic/jest-circus-allure-reporter build && yarn workspace @automattic/calypso-e2e build",
 		"test": "yarn jest",
+		"metrics": "yarn jest --config jest.metrics.config.js",
```

Comment:
Does it make sense to expose this to a script at the root level of the project? `yarn run performance-metrics` or something

Reply:
I don't know, you know better than me, Personally I just tried to follow how e2e tests already work on calypso.

Reply:
I think we're fine, we can always add it later if necessary 👍